src/buffer_manager.ts

```diff
@@ -545,9 +545,12 @@ export class BufferManager implements Disposable {
 
         const text = document.getText();
         const bytes = new TextEncoder().encode(text);
-        await workspace.fs.writeFile(saveUri, bytes);
-        const doc = await workspace.openTextDocument(saveUri);
-        await window.showTextDocument(doc);
+        try {
+            await workspace.fs.writeFile(saveUri, bytes);
+            window.setStatusBarMessage(`Saved "${path.basename(saveUri.fsPath)}"`, 3000);
+        } catch (error) {
+            window.showErrorMessage(`Failed to save ${saveUri.fsPath}: ${error}`);
```

Comment:
```suggestion
            window.showErrorMessage(`Failed to save "${path.basename(saveUri.fsPath)}": ${error}`);
```

Reply:
I think it's better to display the full path in the error message.